topo_processor/metadata/item.py

```diff
@@ -26,3 +27,4 @@ def __init__(self, path: str, collection: Collection):
             properties={},
             stac_extensions=[],
         )
+        get_log().info("Item initialised", path=self.path)
```

Comment:
Once its been created its not actually initalised so this log is a little  missleading, I generally would avoid having logs in constructors too.

Reply:
why would you avoid having logs in constructors?